src/me/eglp/gv2/util/base/guild/config/GuildRemindersConfig.java

```diff
@@ -37,7 +37,7 @@ public class GuildRemindersConfig {
 
 	private GraphiteGuild guild;
 
-	List<GuildReminder> tempGuildReminders = new ArrayList<>();
+	List<GuildReminder> privateVariable = new ArrayList<>();
```

Comment:
This name is even worse

Reply:
Also, the variable is not even `private`

Reply:
> Also, the variable is not even `private`

good point...